cfn-resources/private-endpoint-aws/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
```

Comment:
I believe this would be more relevant if placed in existingprivate endpoint resource, as users that are making use of that resource will likely want to know why it was deprecated.